MonMaraicher/Features/Map/MapViewService.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MapViewService.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 15/03/2024.
+//
+
+import Foundation
+
+final class MapViewService {
+
+    var allFarmers: [Farmer] = []
+
+    func loadFarmers() {
```

Comment:
Why not returning the data directly and avoid having `var allFarmers: [Farmer] = []` above?